iobio_tabs.js

```diff
@@ -0,0 +1,108 @@
+import { getDataBroker } from "./common.js";
+const tabsTemplate = document.createElement('template');
+tabsTemplate.innerHTML = `
+<style>
+:host {
+    --data-color: var(--iobio-data-color, #2d8fc1);
+}
+.iobio-tabs {
+    display: flex;
+    flex-direction: column;
+}
+.tabs {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+}
+.tab {
+    cursor: pointer;
+    text-align: center;
+    color: grey;
+}
+.tab.active {
+    color:  var(--data-color);
+}
+
+.content-container {
+    position: relative;
+    width: 100%;
+    height: 100%;
+}
+
+::slotted(iobio-histogram) {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+}
+</style>
+    <div class="iobio-tabs">
+        <div class="tabs">
+            <span class="tab tab1"></span>  |
```

Comment:
These should probably be named something a bit more descriptive, like `tab1-label` and `tab2-label`